nss-day-cohort-39/tabloid-cli-the-things-from-another-world
# Journal Feature

### Changes
- Added Journal model, repository, and menu manager
- Implemented functionality for listing, adding, editing, and removing journal entries
- Minor adjustments to styling when outputting the list of journal entries (added white space for readability). 


### Testing
- Confirm that Journal Management is the first option in the main menu and you can navigate to the Journal Menu.
- List all journal entries
- Add new journal entry
- Edit an existing journal entry
- Remove a journal entry
- Navigate back to main menu

#### Closes
Closes #3 / Closes #4 / Closes #5 / Closes #6
